src/pyiron_lammps/output_raw.py

```
@@ -0,0 +1,251 @@
+                c2 = np.fromstring(f.readline(), dtype=float, sep=" ")
+                c3 = np.fromstring(f.readline(), dtype=float, sep=" ")
+                cell = np.concatenate([c1, c2, c3])
+                dump.cells.append(to_amat(cell))
+
+            elif "ITEM: NUMBER OF ATOMS" in line:
+                n = int(f.readline())
+                dump.natoms.append(n)
+
+            elif "ITEM: ATOMS" in line:
+                # get column names from line
+                columns = line.lstrip("ITEM: ATOMS").split()
```

Ruff (0.14.10)

Using `.strip()` with multi-character strings is misleading

(B005)